atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -74,6 +148,11 @@ def run(self):
                                  registry_noschema)
                 continue
 
+            if push_conf_registry.insecure:
+                # override insecure if passed
+                insecure = push_conf_registry.insecure
```

Comment:
I didn't run it locally myself, I just read what landscape.io reported (see the 'Show all checks' link below).